test/ring_logger/client_test.exs

```diff
@@ -2,12 +2,12 @@ defmodule RingLogger.Client.Test do
   use ExUnit.Case, async: false
   alias RingLogger.Client
 
-  describe "configure a client at runtime" do
-    setup do
-      {:ok, client} = Client.start_link()
-      {:ok, %{client: client}}
-    end
+  setup do
+    {:ok, client} = Client.start_link()
+    {:ok, %{client: client}}
+  end
```

Comment:
This part fixes the build, right? Could you PR it separately, since I'm wondering if the client vs. server config is going to prompt discussion and more thought.

Reply:
No. This just separates out so I don't have to setup again for a different test. I'm PR fixing the build right now.